cmd/did-rest/startcmd/test/config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "rules": [
+    {
+      "pattern": "did:trustbloc:testnet.trustbloc.local:.+$",
```

Comment:
out of curiosity, did the same pattern as in the original config.json work?

with brackets and $1

Reply:
$1 didn't work - had to be $0 (because it is index)-  I am currently making a change to be closer to original pattern